pymatgen/io/vasp/outputs.py

```diff
@@ -1985,6 +1985,9 @@ def __init__(self, filename):
         run_stats["cores"] = None
         with zopen(filename, "rt") as f:
             for line in f:
+                if "serial" in line:
+                    run_stats["cores"] = 1
+                    break
                 if "running" in line:
```

Comment:
Why is this VASP-related file being modified? Was this just leftover from a `git pull`?

Reply:
No it's not from a git pull, it's actually a little thing that I have added because I once compiled a serial version of VASP (no mpi enabled) and this related pymatgen code broke in such a case. It's so small that I thought I could include it here, but maybe a separate PR is better?

Reply:
Ah I see. Well, thank you for the enhancement! Yes, for clarity I think it would be best to make a separate small PR for this so it doesn't get lost. I know it's a bit of extra work but it will help the maintainers I think (and likely be merged quickly)